packages/atomic/src/pages/examples/ipx.html

```diff
@@ -154,6 +161,10 @@
         display: none;
       }
 
+      atomic-ipx-tabs {
+        width: 100%;
+      }
+
```

Comment:
We will have to keep in mind to update those styles in the SIS builder as well when updating the atomic version